pytype/tools/analyze_project/parse_args.py

```diff
@@ -9,6 +10,20 @@
 
 _ARG_PREFIX = '--'
 
+def _auto_detect_cpus():
+  try:
+    return len(os.sched_getaffinity(0))
+  except AttributeError:
+    return os.cpu_count()
+
+
+def parse_jobs(s):
+  """Parse the --jobs option"""
+  if s == 'auto':
+    n = _auto_detect_cpus()
+    return n if n else 1
+  elif s is not None:
+    return int(s)
```

Comment:
Could you add a "return None" at the end of this method to make it obvious that it intentionally defaults to returning None?